src/Indice.Features.Cases.App/src/app/features/dashboard/dashboard.component.html

```diff
@@ -1,25 +1,27 @@
 <lib-view-layout title="Καλώς ήλθατε, {{ user?.profile?.name }}!"
                  [meta-items]="metaItems">
     <div class="cards-deck-1 mb-4">
-        <div class="cards-deck-3">
-            <app-canvas-tile [title]="'Συνολικες Αιτησεις ανα Τυπο Αιτησης'"
+        <div class="grid grid-cols-3 gap-x-5 gap-y-0">
+            <app-canvas-tile *ngIf="showCanvas(reportTag.GroupedByCasetype)"
```

Comment:
https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496

Για βάλε ένα console log στην showCanvas και θα δεις (λογικά) ότι καλείται διαρκώς.

Πρόταση: αυτό είναι ένα setting που δεν αλλάζει στο runtime της εφαρμογής. Δεν μου φαίνεται άσχημο να καλείται 1 φορά (ανά if) στην onInit του component και τα αποτελέσματα της να αποθηκεύονται σε μεταβλητές, οι οποίες θα συνδεθούν στα ngIfs του template (Strategy 2— Manually calculate the values του επισυναπτόμενου άρθρου).